crowbar_framework/app/helpers/tag_helper.rb

```diff
@@ -18,31 +18,28 @@
 module TagHelper
   # Convert a hash into a multidimensional unordered list
   def hash_to_ul(hash)
-    content_tag(
-      :ul,
-      [].tap do |output|
-        hash.each do |key, value|
-          content = []
-
-          if key.is_a? Hash
-            content.push hash_to_ul(key)
-          else
-            content.push content_tag(:em, key)
-          end
+    content_tag :ul do
+      hash.collect do |key, value|
+        content = []
 
-          if value.is_a? Hash
-            content.push hash_to_ul(value)
-          else
-            content.push value == nil ? "" : ": #{value}"
-          end
+        if key.is_a? Hash
+          content.push hash_to_ul(key)
+        else
+          content.push content_tag(:em, key)
+        end
 
-          output.push content_tag(
-            :li,
-            content.join("\n")
-          )
+        if value.is_a? Hash
+          content.push hash_to_ul(value)
+        else
+          content.push value == nil ? "" : ": #{value}"
```

Comment:
Style/NilComparison: Prefer the use of the `nil?` predicate. (https://github.com/bbatsov/ruby-style-guide#predicate-methods)